### PR TITLE
Add page transporter and web element actions

### PR DESCRIPTION
**Pull request by danielaSantaCruz**

### CODE REVIEW

**Review by edsonarios — APPROVED** [review]
Good job

File: src/main/java/core/selenium/WebElementAction.java [hunk 1]
```diff
@@ -15,7 +15,7 @@
 
     public WebElementAction() {
         driver = DriverManager.getInstance().getDriver();
-        wait = new WebDriverWait(driver, 30);
+        wait = new WebDriverWait(driver, 40);
```
Comment (cristianfchq) [inline src/main/java/core/selenium/WebElementAction.java]: Remove magic numbers

File: src/test/java/base/BaseTest.java [hunk 2]
```diff
@@ -1,30 +1,36 @@
...
         driverManager = DriverManager.getInstance();
         loginPage = pageTransporter.navigateToLoginPage();
+        homePage = loginPage.loginSuccessful(getEnvVariables("USER"), getEnvVariables("PASSWORD"));
+        legalEntitiesPage = pageTransporter.navigateToLegalEntityPage();
     }
 
-    @AfterClass
+    @AfterTest
     public void tearDown() {
+//        legalEntitiesPage.clickDeleteBtn();
```
Comment (cristianfchq) [inline src/test/java/base/BaseTest.java]: maybe you can remove this comment

Comment (edsonarios) [inline src/test/java/base/BaseTest.java]: This comment should be deleted

File: src/test/java/legalentity/NewLegalEntityTests.java [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    String dropdownOption = "Active";
+    String cityStatePostalCode = city + ", " + state + " " + postalCode;
+
+    @Test
+    public void testCreateLegalEntityWithRequiredFields() {
+        newLegalEntityPage = legalEntitiesPage.clickNewBtn();
+        legalEntityPage = newLegalEntityPage.setInputField("name", name).clickSaveBtn();
+        String expectedMessage = legalEntityPage.getUserSuccessMessage();
+        String expectedName = legalEntityPage.getEntityNameText(name);
+        //Verify success message
```
Comment (edsonarios) [inline src/test/java/legalentity/NewLegalEntityTests.java]: This comment should be deleted

File: src/test/java/legalentity/NewLegalEntityTests.java [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+    public void testCreateLegalEntityWithFullFields() {
+        newLegalEntityPage = legalEntitiesPage.clickNewBtn();
+        legalEntityPage = newLegalEntityPage.setInputField("name", nameComplete)
+                .setInputField("company name", company).setStreetTxtBox(street)
+                .setDescriptionTxtBox(description).setInputField("city", city)
+                .setInputField("state", state).setInputField("postal code", postalCode)
+                .setInputField("country", country).selectFromDropDown("select", dropdownOption)
+                .clickSaveBtn();
+        String expectedMessage = legalEntityPage.getUserSuccessMessage();
+        //Verify success message
```
Comment (edsonarios) [inline src/test/java/legalentity/NewLegalEntityTests.java]: This comment should be deleted

File: src/test/java/login/LoginTests.java [hunk 5]
```diff
@@ -8,10 +8,8 @@
...
 public class LoginTests extends BaseTest {
 
-    @Test
-    public void testLoginInTheWebPage() {
-        HomePage homePage = loginPage.setUserName(getEnvVariables("USER"))
-                .setPassword(getEnvVariables("PASSWORD"))
-                .clickLoginBtn();
-    }
+//    @Test
+//    public void testLoginInTheWebPage() {
```
Comment (edsonarios) [inline src/test/java/login/LoginTests.java]: This comment should be deleted

**Review by cristianfchq — APPROVED** [review]

File: src/main/java/core/selenium/WebElementAction.java (hunk 1 above)
Comment (danielaSantaCruz) [reply src/main/java/core/selenium/WebElementAction.java]: Already done

File: src/test/java/base/BaseTest.java (hunk 2 above)
Comment (danielaSantaCruz) [reply src/test/java/base/BaseTest.java]: okay, It's already done

Comment (danielaSantaCruz) [reply src/test/java/base/BaseTest.java]: okay, It's already done

File: src/test/java/login/LoginTests.java (hunk 5 above)
Comment (danielaSantaCruz) [reply src/test/java/login/LoginTests.java]: okay, It's already done

**Review by edsonarios — APPROVED** [review]

**Pull request merged**